csrc/ops/alias.cpp

```diff
@@ -22,10 +22,20 @@ Val* set(Val* v) {
   return out;
 }
 
+Val* set(Val* v, CacheOp cache_op) {
```

Comment:
How about giving `cache_op` here the default value `CacheOp::Unspecified` and remove the other flavor of `set` that doesn't take `cache_op`?